### PR TITLE
Revert "Mybranch5"

### PR DESCRIPTION
**Pull request by endeavor2299**
dfaf

### CODE REVIEW

**Review by endeavor66 — APPROVED** [review]
I approved

File: mybranch4 [hunk 1]
```diff
@@ -1,2 +0,0 @@
-t
-hello mybranch4
```
Comment (endeavor66) [inline mybranch4:2]: why

**Review by weigangNJU — APPROVED** [review]
i approve

**Review by endeavor66 — APPROVED** [review]

**Review by endeavor66 — CHANGES_REQUESTED** [review]
a

**Review by endeavor66 — COMMENTED** [review]
well done

**Review by endeavor66 — COMMENTED** [review]
ok

File: mybranch5 [hunk 2]
```diff
@@ -1,5 +0,0 @@
-public static void main(){
-    int a = 1;
-    int b = a + 1;
-    System.out.println("hello, world");
```
Comment (endeavor66) [inline mybranch5:4]: ook

File: mybranch5 [hunk 3]
```diff
@@ -1,5 +0,0 @@
-public static void main(){
```
Comment (endeavor66) [inline mybranch5:1]: dafxc

Comment (endeavor66) [reply mybranch5:1]: kjkdfa

**Review by endeavor66 — COMMENTED** [review]
yes

**Review by endeavor66 — DISMISSED** [review]

**Review by endeavor66 — DISMISSED** [review]

**Review by endeavor66 — DISMISSED** [review]
daf

**Comment by endeavor66** [comment]
okok

**Comment by endeavor66** [comment]
okok

**Pull request closed without merging**